File: PROYECTO_RESTFULL_JAVA_BDD/01 SERVIDOR/ComercializadoraElectrodomesticos/src/main/java/ec/edu/monster/model/Factura.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package ec.edu.monster.model;

import java.util.Date;
class Factura {
    private int idFactura;
    private String numeroFactura;
    private String cedulaCliente;
    private String nombreCliente;
    private Date fechaFactura;
    private double subtotal;
    private double descuento;
    private double total;
    private String formaPago;
    private String estado;
    
    public Factura() {}
    
    // Getters y Setters
    public int getIdFactura() { return idFactura; }
    public void setIdFactura(int idFactura) { this.idFactura = idFactura; }
    
    public String getNumeroFactura() { return numeroFactura; }
    public void setNumeroFactura(String numeroFactura) { this.numeroFactura = numeroFactura; }
    
    public String getCedulaCliente() { return cedulaCliente; }
    public void setCedulaCliente(String cedulaCliente) { this.cedulaCliente = cedulaCliente; }
    
    public String getNombreCliente() { return nombreCliente; }
    public void setNombreCliente(String nombreCliente) { this.nombreCliente = nombreCliente; }
    
    public Date getFechaFactura() { return fechaFactura; }
    public void setFechaFactura(Date fechaFactura) { this.fechaFactura = fechaFactura; }
    
    public double getSubtotal() { return subtotal; }
    public void setSubtotal(double subtotal) { this.subtotal = subtotal; }
    
    public double getDescuento() { return descuento; }
    public void setDescuento(double descuento) { this.descuento = descuento; }
    
    public double getTotal() { return total; }
    public void setTotal(double total) { this.total = total; }
    
    public String getFormaPago() { return formaPago; }
    public void setFormaPago(String formaPago) { this.formaPago = formaPago; }
    
    public String getEstado() { return estado; }
    public void setEstado(String estado) { this.estado = estado; }
}

File: EXAMEN_SOAP_DOTNET_BDD/01 SERVIDOR/02_SERVIDOR_COMERCIALIZADORA/Comercializadora_Soap_Dotnet/ec.edu.monster.modelo/Factura.cs
using System;
using System.Runtime.Serialization;

namespace Comercializadora_Soap_Dotnet.ec.edu.monster.modelo
{
    [DataContract(Name = "factura")]
    public class Factura
    {
        [DataMember(Order = 1)]
        public int IdFactura { get; set; }

        [DataMember(Order = 2)]
        public string NumeroFactura { get; set; }

        [DataMember(Order = 3)]
        public string CedulaCliente { get; set; }

        [DataMember(Order = 4)]
        public string NombreCliente { get; set; }

        [DataMember(Order = 5)]
        public DateTime FechaFactura { get; set; }

        [DataMember(Order = 6)]
        public decimal Subtotal { get; set; }

        [DataMember(Order = 7)]
        public decimal Descuento { get; set; }

        [DataMember(Order = 8)]
        public decimal Total { get; set; }

        [DataMember(Order = 9)]
        public string FormaPago { get; set; }

        [DataMember(Order = 10)]
        public string Estado { get; set; }

        public Factura() { }
    }
}